.github/workflows/ci.yml

```
@@ -21,7 +21,7 @@ jobs:
         with:
           ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
       - name: Install dependencies
-        run: npm ci
+        run: npm install
```

`npm install` changes the lock file. It should be avoided
```suggestion
        run: npm ci
```